src/Microsoft.Health.Fhir.Shared.Api/Features/Filters/OperationOutcomeExceptionFilterAttribute.cs

```diff
@@ -202,12 +211,7 @@ public override void OnActionExecuted(ActionExecutedContext context)
                         healthExceptionResult = CreateOperationOutcomeResult(microsoftHealthException.Message, OperationOutcome.IssueSeverity.Error, OperationOutcome.IssueType.Invalid, HttpStatusCode.RequestHeaderFieldsTooLarge);
                         break;
                     default:
-                        healthExceptionResult = new OperationOutcomeResult(
-                            new OperationOutcome
-                            {
-                                Id = _fhirRequestContextAccessor.RequestContext.CorrelationId,
-                            },
-                            HttpStatusCode.InternalServerError);
+                        healthExceptionResult = CreateOperationOutcomeResult(microsoftHealthException.Message, OperationOutcome.IssueSeverity.Error, OperationOutcome.IssueType.Unknown, HttpStatusCode.InternalServerError);
```

Comment:
>(microsoftHealthException.Message [](http://example.com/codeflow?start=76&length=33)

I tend to think there was a reason to hide internal guts and not show them to an user.  Any reason why we want to show message nowadays?

Reply:
That is a fair point. I made this change for internal consistency, but I can alter the CreateOperationOutcomeResult method to just make message optional.

Reply:
Used string.empty instead, less refactoring.